appeals/web/src/server/appeals/appeal-details/site-visit/__tests__/site-visit.test.js

```diff
@@ -311,12 +334,16 @@ describe('site-visit', () => {
 			nock.cleanAll();
 		});
 
-		it('should render the visit scheduled confirmation page', async () => {
-			const response = await request.get(`${baseUrl}/1${siteVisitPath}${visitScheduledPath}`);
-			const element = parseHtml(response.text);
+		for (const confirmationPageTypeToRender of ['new', 'unchanged', 'visit-type', 'date-time']) {
+			it(`should render the visit scheduled/rescheduled confirmation page of pageType "${confirmationPageTypeToRender}"`, async () => {
```

Comment:
As a side note to the point about not having logic in tests, interpolating `confirmationPageTypeToRender` into the test name makes the resulting snapshot a bit confusing (for example, when reading the snapshots, it wasn't immediately clear what `site-visit GET /site-visit/visit-scheduled/:confirmationPageTypeToRender should render the visit scheduled/rescheduled confirmation page of pageType "date-time"` meant). In my opinion it should be easy to figure out if a given snapshot is correct by reading it's name/title and looking at the associated rendered HTML, so in this case I'd suggest something like `should render the visit changed confirmation page with updated time and date`